### ISSUE TITLE
Rating Zero Stars

### ISSUE THREAD

**Issue by pfefferminzz** [ludumdare/ludumdare#972]
I think it would be good to be able to rate zero stars in a category to show the author he should have opted out of this category without giving one star. No rating is not the same as rating no star. Mockup:

![nostars](https://cloud.githubusercontent.com/assets/5459440/26274535/3d12e620-3d4c-11e7-8385-85408afa4c4c.png)

**Comment by Cerno-b** [comment]
*fistbump*

**Comment by pfefferminzz** [comment]
I already referenced it in your issue. I made this mockup before reading it, though. :)

**Comment by Cerno-b** [comment]
Yeah, I only added it here for two-way linking reasons, or does Github do this automatically?

**Comment by pfefferminzz** [comment]
Yes, see above: pfefferminzz referenced this issue an hour ago. Report missing opt-out #968

**Comment by pfefferminzz** [comment]
Hmmm.

**Comment by Cerno-b** [comment]
Ah, there it is, I totally overlooked that.

**Comment by local-minimum** [comment]
If we have a way of handling such data internally it would be nice warning on peoples stats to have a warning triangle with text `Forgot to opt-out?` or something if enough people have selected this? I still think it should be the person making the game who decides about opt-out, possibly Mike/moderators but not the community.

**Comment by zwrawr** [comment]
I don't like the suggested implementation of this idea, is seems needlessly complicated.

But i do like the idea. I think it could be done much simpler.
If a game doesn't have music then don't rate it in music. 

Then the site can pop up a flag to the author if the number of ratings on a specific category is much lower than the avg number of ratings per category.

**Comment by pfefferminzz** [comment]
Popping up a flag to the author would already be possible.

My suggestion points at the difference between not rating a category (for unknown reasons) and rating zero on purpose (opt-out indication). The symbol could look like a report button, i.e. **/!\**. But maybe i'm really thinking too complicated.

**Comment by zwrawr** [comment]
I just don't think is necessary, if we can already detect it implicitly from the way users vote. I don't think we need an explicit button. And if we do need a button it should be less prominent, in a drop down list with a small icon on the corner of the game page. It shouldn't be `easy` to accidentally report a any thing.

**Comment by pfefferminzz** [comment]
I wanted an explicit button, because it could still be counted as a rating in this category. http://vatcss.info/ld38votes/ shows that Audio, Humor and Mood tend to be categories with less votes. The report was meant to be an opt-out-indicator to the author in the first place.

But you are right, accidental reporting would be a newly introduced problem i did not think of.

**Comment by Cerno-b** [comment]
So would this be a helper for the creator or should it be a bit more pushy? 

What about people who didn't understand the rules properly and try to get rated on music they downloaded? I can imagine that beginners who make an effort but only create basic music will be upset if they get a lower score than someone with awesome music they didn't write.

**Comment by zwrawr** [comment]
The people could go into a drop down menu and report it. Which would notify the games author and mod's who can check on a case by case basis and decide weather on not it seems right.

Obviously the mod stuff would have to wait till the site has mods but it will do eventually.

**Comment by Cerno-b** [comment]
Yeah, that would work. I didn't know what role Mods will play in the future and how much of a burden they will be supposed to bear (I guess being spammed by reports like these can be quite a mouthful). So if moderated opting-out would work, that would probably be the simplest solution.

**Comment by quantumrain** [comment]
If we did use a design similar to the image in the first post of this thread, one way to reduce the likelihood of accidental reports would be to throw up a popup if the user clicks it.

It could ask:

```Do you want to suggest to the author of this game that they should opt of the XXX category?```

Then if they click "Yes" we take them through whatever standard report a game UI we have- otherwise it just acts like a clear this rating button; so it essentially just becomes a shortcut to reporting a game.
